.infrastructure/busybox.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+    name: busybox
+    namespace: todoapp
+spec:
+    containers:
+    - name: busybox
+      image: parthyadav/busyboxplus:curl-v22.12.17
```

Comment:
This is a wrong image. Use radial/busyboxplus:curl

Reply:
When I tried to use image "radial/busyboxplus:curl" I had the next problem:
__Failed to pull image "radial/busyboxplus:curl": [DEPRECATION NOTICE] Docker Image Format v1 and Docker Image manifest version 2, schema 1 support is disabled by default and will be removed in an upcoming release. Suggest the author of docker.io/radial/busyboxplus:curl to upgrade the image to the OCI Format or Docker Image manifest v2, schema 2. More information at https://docs.docker.com/go/deprecated-image-specs/__
That's why I used the other imege, newer.
I think using the other image it is not a problem. Main goal busybox consist of connecting to pod with application and execute curl command. This image resolve this problem.